iot/dotnet-iot-and-nanoframework/README.md

```diff
@@ -0,0 +1,67 @@
+--- 
+languages:
+- csharp
+products:
+- dotnet
+- dotnet-core
+- azure
+- azure-iot-hub
+page_type: sample
+name: "Compare .NET nanoFramework with .NET Core IoT"
```

Comment:
I guess it's .NET IoT, you can remove the Core everywhere.

Reply:
Thanks @Ellerbach this is fixed